modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -149,7 +149,12 @@ def build_claim_structure(data:, lighthouse_id:, upstream_id:)
             status: detect_status(data),
             submitter_application_code: data[:submtr_applcn_type_cd],
             submitter_role_code: data[:submtr_role_type_cd],
-            '5103_waiver_submitted'.to_sym => map_yes_no_to_boolean('filed5103_waiver_ind', data[:filed5103_waiver_ind])
+            '5103_waiver_submitted'.to_sym => map_yes_no_to_boolean('filed5103_waiver_ind',
+                                                                    data[:filed5103_waiver_ind]),
+            jurisdiction: data[:regional_office_jrsdctn],
+            maxEstClaimDate: data[:maxEstClaimDate],
```

Comment:
According to [this discussion with Matt](https://lighthouseva.slack.com/archives/C03P95NJZ2T/p1659461276720939), we'll need to pull this data from `data[:max_est_claim_complete_dt]` and it looks like that attribute is conditionally provided by BGS.